.github/workflows/ci.yml

```diff
@@ -73,7 +125,7 @@ jobs:
         echo "deb-src http://archive.ubuntu.com/ubuntu/ $(lsb_release -cs) main restricted universe multiverse" | sudo tee -a /etc/apt/sources.list
         sudo apt update
         sudo apt build-dep systemd
-        git clone https://github.com/systemd/systemd --depth=1 && cd systemd
+        git clone https://github.com/systemd/systemd --depth=1 --branch=v248 && cd systemd
```

Comment:
That doesn't look right. Either use git tip (`main`) on the assumption that it's the latest and greatest, or the stable branch, systemd-stable/v248-stable in this case. But that raises the problem how to keep updating this link when the next release happens. A given git tag might not even compile once enough time has passed (usually because of kernel and glibc header changes, but there are other reasons sometimes).

Reply:
> That doesn't look right. Either use git tip (`main`) on the assumption that it's the latest and greatest,

kept hitting [22](https://github.com/systemd/systemd/issues/19442) on that assumption :)

Reply:
There's been some work going on to fix the connection timed out error in the systemd repo. Can you try removing this again to see if we still hit that error?